util/util.go

```diff
@@ -230,3 +230,13 @@ func GetTransitGatewayLocationAndRouting(powerVSZone *string, vpcRegion *string)
 	// since VPC region is not set and used PowerVS region to calculate the transit gateway location, hence returning local routing as default.
 	return &location, ptr.To(false), nil
 }
+
+// VPCZonesForVPCRegion returns the VPC zones associated with the VPC region.
+func VPCZonesForVPCRegion(region string) ([]string, error) {
+	for pvsRegion, regionalComponentsData := range Regions {
+		if regionalComponentsData.VPCRegion == region {
+			return Regions[pvsRegion].VPCZones, nil
```

Comment:
we can directly return regionalComponentsData.VPCZones?

Reply:
Comment addressed, thanks!